src/security/ServerOptions.h

```diff
@@ -35,7 +35,6 @@ class ServerOptions : public PeerOptions
         // is more secure to have only a small set of trusted CA.
         flags.tlsDefaultCa.defaultTo(false);
     }
-    ServerOptions(const ServerOptions &) = default;
```

Comment:
If copying is needed, you may be able to trivially implement the copy constructor via the existing copy assignment operator:

```C++
ServerOptions(const ServerOptions &other): ServerOptions() { *this = other; }
```

Reply:
hmm, github wont let me apply this change suggestion directly. But done.